### PR TITLE
Acorn parse options (fix for PR#8)

### PR DESCRIPTION
**Pull request by kjoshi1988**
Fix for PR #8 
- Added parse options support
- Removed phantomJS as test runner in CI as it is already deprecated and it  was causing test failure.
- Added Chromium as the test runner.

### CODE REVIEW

**Comment by aghassemi** [comment]
@krisselden as discussed, we still need `ember-cli-optimize` and this PR will allow us to enable `await/async`. Please take a look and would appreciate a new release with this fix. Thanks!

@kjoshi1988 Please add the issue number this resolves in the description of the PR.( i.e. Closes #8 )

File: ember-cli-build.js [hunk 1]
```diff
@@ -138,6 +138,9 @@ module.exports = function(defaults) {
         'ember-resolver/utils/create':true,
         'ember-resolver/utils/make-dictionary':true,
         'ember-resolver/utils/module-registry':true
+      },
+      parseOptions: {
+        ecmaVersion: 8
```
Comment (aghassemi) [inline ember-cli-build.js]: should we use year-base versions (e.g. 2017 for 9, 2018 for 11?) instead? it's more descriptive that 8, 9, etc..

Comment (kjoshi1988) [reply ember-cli-build.js]: Good point, will make the change.

File: testem.js [hunk 2]
```diff
@@ -4,10 +4,19 @@ module.exports = {
...
   ],
   "launch_in_dev": [
-    "PhantomJS",
     "Chrome"
-  ]
+  ],
+  "browser_args": {
+    "Chrome": [
+      "--headless",
+      "--disable-gpu",
```
Comment (krisselden) [inline testem.js:15]: you shouldn't need disable-gpu, chrome should automatically fallback to software rendering if there is no GPU

**Review by krisselden — APPROVED** [review]

**Pull request merged**